CosNet.API/Data/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -82,8 +82,8 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasColumnType("uniqueidentifier");
 
                     b.Property<string>("Description")
-                        .HasColumnType("nvarchar(150)")
-                        .HasMaxLength(150);
+                        .HasColumnType("nvarchar(500)")
```

Comment:
Why has this moved up to 500?

Reply:
Check the designs there it said that description should be 500 chars long